conda-forge.yml

```
@@ -3,5 +3,6 @@ appveyor:
 compiler_stack: comp7
 max_py_ver: '37'
 max_r_ver: '35'
+provider: {linux_aarch64: azure, linux_ppc64le: azure}
```

```suggestion
provider: {linux_aarch64: azure, linux_ppc64le: default}
```
ppc64le builds fail on azure. They need to run on travis (`default`, or `travis`)

---

actually they're not failing anymore, so we might not need to